myRental/modules/database.js

```diff
@@ -10,24 +10,27 @@ let Log = require('./Log.js');
 
 let logger = new Log();
 let User = null;
```

Comment:
You don't need to be setting `User` or `Request` to null. By default, a let is undefined which should work with your current implementation.